lib/datastore/transaction.js

```diff
@@ -85,16 +85,18 @@ nodeutil.inherits(Transaction, DatastoreRequest);
  *     a transaction.
  *
  * @example
- * transaction.begin(function(err) {
- *   // Perform Datastore operations as usual.
- *   transaction.get(dataset.key(['Company', 123]), function(err, entity) {
- *     // Commit the transaction.
- *     transaction.finalize(function(err, apiResponse) {});
+ * dataset.runInTransaction(function(transaction, done) {
+ *   transaction.begin(function(err) {
```

Comment:
Remove `transaction.begin` from these examples. Begin is called automatically when `dataset.runInTransaction` is called.